templates/orgs/org_send_invite.haml

```diff
@@ -0,0 +1,19 @@
+-extends 'smartmin/form.html'
+-load smartmin
+-load i18n
+
+-block fields
+    -render_field 'email'
+    -render_field 'user_group'
```

Comment:
`margin-top: 10px` would be welcome on this field's label

Reply:
Have added styles for user_group label.